### ISSUE TITLE
Dropped Sound

### ISSUE THREAD

**Issue by onetimesignup** [EchoDev/TwitchHTML5#15]
For some reason yesterday when I watched a `splattercatgaming` all sound was gone, I tried switching from source to medium without it having any effect however jumping directly to the html5 page worked like normal.

Flash didn't have any issues and people thought I was trolling, derp.
I hadn't muted the stream and viewing other streams worked fine.

**Comment by EchoDev** [comment]
I follow that channel, when it goes online I will check if it works for me.

**Comment by onetimesignup** [comment]
I'm pretty sure it had something to do with the dissapearing "mature streamer" box, because like i said, the second I got it to show on the html5 page and clicked on it sound worked like normal everywhere.

Twitch also had issues when I encountered this so they may have resolved the problem, who knows.

**Comment by EchoDev** [comment]
I don't have any problems with that streamer atm with a clean browser (no cache/cookies)

**Comment by onetimesignup** [comment]
yup i just checked him out now as well and i got the "mature stream" popup like normal, just saying last time it didnt show but the video played just not without audio, earlier version of the twitch "mature" blocker thingy used to behave like that, ie the stream would run in the background but hidden by the "mature streamer" popup and the audio would be muted so it wouldnt suprice me if this was just a bug with that popup.

**Comment by EchoDev** [comment]
Do you still have this problem? I haven't noticed this problem yet so can't say anything about it atm.

**Comment by onetimesignup** [comment]
It happens when the 'mature streamer' box doesnt display which seems to be occuring more frequently when twitch is having site issues and cant for instance display live streamers or having issues listing followers.

**Comment by EchoDev** [comment]
Yeah I have been reading people are having issues on the Twitch sites. Even streamers are complaining their stream is showing as offline. Hopefully Twitch can make some progress soon with their own HTML5 player.

**Comment by onetimesignup** [comment]
Would it be possible to offer a option to force-accept the 'mature streamer'  check ?
Like a checkbox next to the theatre button ?

It would be super handy.

**Comment by EchoDev** [comment]
I can look into it, I can maybe add a button next to the share button. It might take me a few days to get on it tho.
